modules/olm-accessing-images-private-registries.adoc

```diff
@@ -0,0 +1,159 @@
+====
+
+.. It is recommended to include credentials for only one registry per secret, and manage credentials for multiple registries in separate secrets. Multiple secrets can be included in a `CatalogSource` object in later steps, and {product-title} will merge the secrets into a single virtual credentials file for use during an image pull.
++
+A registry credentials file can, by default, store details for more than one registry. Verify the current contents of your file. For example:
++
+.File storing credentials for two registries
+[source,json]
+----
+{
+        "auths": {
+                "registry.redhat.io": {
+                        "auth": "FrNHNydQXdzclNqdg=="
+                },
+                "quay.io": {
+                        "auth": "Xd2lhdsbnRib21iMQ=="
+                }
+        }
+}
+----
++
+Because this file is used to create secrets in later steps, ensure that you are storing details for only one registry per file. This can be accomplished by using either of the following methods:
++
+--
+* Use the `podman logout <registry>` command to remove credentials for additional registries until only the one registry you want remains.
+* Edit your registry credentials file and separate the registry details to be stored in multiple files. For example:
++
+.File storing credentials for one registry
+[source,json]
+----
+{
+        "auths": {
+                "registry.redhat.io": {
+                        "auth": "FrNHNydQXdzclNqdg=="
+                }
+        }
+}
+----
++
+.File storing credentials for another registry
+[source,json]
+----
+{
+        "auths": {
+                "quay.io": {
+                        "auth": "Xd2lhdsbnRib21iMQ=="
+                }
+        }
+}
+----
+--
+
+.. Create a secret that contains the authentication credentials for a private registry:
++
+[source,terminal]
+----
+$ oc create secret generic <secret_name> \
+    --from-file=.dockerconfigjson=<path/to/registry/credentials> \
+    --type=kubernetes.io/dockerconfigjson
+----
++
+Repeat this step to create additional secrets for any other required private registries, updating the `--from-file` flag to specify another registry credentials file path.
+
+. Create or update an existing `CatalogSource` object to reference one or more secrets:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: my-operator-catalog
+  namespace: openshift-marketplace
+spec:
+  sourceType: grpc
+  secrets: <1>
+  - "<secret_1>"
+  - "<secret_2>"
+  image: <registry>:<port>/<namespace>/<image>:<tag>
+  displayName: My Operator Catalog
+  publisher: <publisher_name>
+  updateStrategy:
+    registryPoll:
+      interval: 30m
+----
+<1> Add a `spec.secrets` section and specify any required secrets.
+
+. If any Operator or Operand images referenced by a subscribed Operator requires access to a private registry, add the authentication details to the global cluster pull secret in the `openshift-config` project.
```

Comment:
I think subject-verb agreement is off here? "requires" is referring to "images", right? Might also help to use the complementizer her for clarity, e.g.
`If any Operator or Operand images that are referenced by a subscribed Operator require access [...]`